.devcontainer/requirements.txt

```diff
@@ -15,4 +15,6 @@ pathvalidate==3.2.0
 svgpathtools==1.6.1
```

Comment:
I've updated the PR validator github action to handle forked repos
can you rebase your branch on the latest main ?

Reply:
It works now 🥳